exec/java-exec/src/main/resources/rest/profile/list.ftl

```diff
@@ -64,6 +65,8 @@
             <td>
               <div style="height:100%;width:100%">${query.getState()}</div>          
             <td>
+              <div style="height:100%;width:100%">${(.now?long - query.getStartTime()) / 1000} sec</div>          
```

Comment:
Good point. For long running queries, reporting in a single unit actually makes readability much harder. Modified in the next commit. Had to modify the UserBitShared class, however, to add this functionality for the query profile. Unless there is a change to the protobuf, we should be fine.